### ISSUE TITLE
Monday as first day of the week

### ISSUE THREAD

**Issue by Rudy96** [mpcjanssen/simpletask-android#676]
In for example Germany the first day of the week is monday and not sunday. It would be nice to select which day should be first day of the week in the settings.

**Comment by smichel17** [comment]
What places would this affect simpletask functionality? The only one I can think of off the top of my head is the date picker, which I feel like is *supposed* to respect your locale settings anyway...

**Comment by mpcjanssen** [comment]
This is/should be locale specific. Simpletask will not have a locale override. @Rudy96 what is your phone language/locale?

**Comment by Rudy96** [comment]
Yes. The calendar is used to select the start date or the due date (after I activated this feature in the settings). I use Android 4.4 and the language is set to german. But the calendar starts with sunday.

**Comment by DuncanLithgow** [comment]
My regional setting is DA-EN (Denmark/english) and start on Monday in Android Oreo (8.0.0). I suggest this issue be closed unless the behavior is confirmed.

**Comment by Rudy96** [comment]
Meanwhile I have changed the Android-Version to 5.1. So I can't check if it works correctly with Version 4.4 now. With my actual Version 5.1 everything is fine. So for me it would be ok if the issue will be closed.

**Comment by RFDAJE** [comment]
I'm using simpletask on miui 11.0.1 android 10 locale zh_CN, which on default set first day of the week as Sunday. in miui calendar app preference allows user to choose the first day of the week, set it as Monday, in simpletask still shown Sunday as the first day.

my question is, could it possible to do an extra query from calendar app setting to determine the first day of the week to respect user setting.
